docker-compose/docker-compose.multiserver.mariadb.yml

```diff
@@ -24,11 +26,16 @@ services:
       - MYSQL_DATABASE=opencast
       - MYSQL_USER=opencast
       - MYSQL_PASSWORD=opencast
-    ports:
-      - "3306:3306"
     volumes:
       - ./assets/opencast-ddl.sql:/docker-entrypoint-initdb.d/opencast-ddl.sql:ro
       - db:/var/lib/mysql
+    command: [
+            '--wait_timeout=28800',
+        ]
```

Comment:
Could you use the more compact version: `command: "--wait_timeout=28800"`

Reply:
Solved